docker/test/fasttest/run.sh

```diff
@@ -259,7 +268,7 @@ TESTS_TO_SKIP=(
     00974_query_profiler
 
     # Look at DistributedFilesToInsert, so cannot run in parallel.
-    01460_DistributedFilesToInsert
+    01457_DistributedFilesToInsert
```

Comment:
But the test name is `01460_DistributedFilesToInsert`, why did you change this?

Reply:
This is some weird mistake, thanks for noticing, I'll fix.